tests/src/providers/testqgseptprovider.cpp

```diff
@@ -355,9 +366,10 @@ void TestQgsEptProvider::testIdentify()
       point[ QStringLiteral( "UserData" ) ] =  "17" ;
       point[ QStringLiteral( "X" ) ] =  "498066.27" ;
       point[ QStringLiteral( "Y" ) ] =  "7050995.06" ;
-      point[ QStringLiteral( "Z" ) ] =  "74.60" ;
+      point[ QStringLiteral( "Z" ) ] =  74.6 ;
```

Comment:
looks strange that is it not string like X a Z in test above

Reply:
With string representation, the tests were failing with LAZ because of this:
```
   Actual   (identifiedPoints[0][k]): QVariant(double,74.60000000000001)
   Expected (expected[0][k])        : QVariant(QString,74.60)
```
Will try to change them all other expected values to numbers as well.

@nyalldawson test for GpsTime is few lines above - I just made the test to be run for both BIN and LAZ encoding of EPT (previously it was only run for BIN encoded EPT)